Makefile

```diff
@@ -1,10 +1,10 @@
 .DEFAULT_GOAL := local
 
 ifndef VERSION
-	VERSION = ${shell git describe --tags --always}
+	VERSION=${shell git rev-parse --abbrev-ref HEAD}-${shell git rev-parse --short HEAD} # branch-commit_hash
 endif
 
-ifeq ($(VERSION),stable)
+ifeq (${shell git rev-list -n 1 tags/stable},${shell git rev-parse HEAD}) # if stable tag is the same as current commit
```

Comment:
Thinking about the release poccess I'm wondering if that could overwrite a custom VERSION.
Maybe move the block also into the `ifndef VERSION` block to be on the safe side. A custom VERSION must win always